packages/TorneloScoresheet/src/pages/ResultDisplay/ResultDisplay.tsx

```diff
@@ -21,8 +22,25 @@ const ResultDisplay: React.FC = () => {
   }`;
 
   const handleEmailGame = (): void => {
-    // TODO: Implement email logic
-    console.log('email game button pressed');
+    const subject =
```

Comment:
Could this also be in the send email file?  That way it's all in the same place and it keeps the result display file purely as the result display page.

Reply:
Agreed, the email function should probably take the email, `ChessGameResult`, and  `ChessGameInfo` as arguments